site/src/pages/TemplatePage/TemplateInsightsPage/IntervalMenu.tsx

```diff
@@ -0,0 +1,87 @@
+import CheckOutlined from "@mui/icons-material/CheckOutlined";
+import ExpandMoreOutlined from "@mui/icons-material/ExpandMoreOutlined";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Menu from "@mui/material/Menu";
+import MenuItem from "@mui/material/MenuItem";
+import { useState, useRef } from "react";
+
+export const insightsIntervals = {
+  day: {
```

Comment:
Since this object's properties seem like they should always be in sync with `ActiveUserChartProps` from `ActiveUserChart, do you think it's worth making that more explicit at the type level?

I guess I'm just worried that if there isn't a centralized source of truth, and because these types aren't defined via the server, there could be accidental drift in the codebase over time

Reply:
I don't think in this case `ActiveUserChart` should be the centralized source of truth since those controls are not directly attached to it. At some point, if we see it is attached to the `ActiveuserChart` we can do it.

Reply:
Okay, makes sense